go.mod

```diff
@@ -3,48 +3,62 @@ module github.com/ibm/the-mesh-for-data
 	github.com/IBM/satcon-client-go v0.0.0-20210107134702-1be071817792
+	github.com/Masterminds/sprig/v3 v3.2.0 // indirect
+	github.com/Masterminds/squirrel v1.5.0 // indirect
 	github.com/Microsoft/go-winio v0.4.15 // indirect
 	github.com/containerd/continuity v0.0.0-20201119173150-04c754faca46 // indirect
+	github.com/form3tech-oss/jwt-go v3.2.2+incompatible // indirect
 	github.com/ghodss/yaml v1.0.0
 	github.com/go-chi/chi v4.1.2+incompatible
 	github.com/go-chi/render v1.0.1
-	github.com/go-logr/logr v0.1.0
-	github.com/gogo/protobuf v1.3.1
-	github.com/golang/protobuf v1.3.5
-	github.com/google/uuid v1.1.2 // indirect
+	github.com/go-logr/logr v0.2.0
+	github.com/gogo/protobuf v1.3.2
+	github.com/golang/protobuf v1.4.3
 	github.com/gregjones/httpcache v0.0.0-20190611155906-901d90724c79 // indirect
 	github.com/hashicorp/vault/api v1.0.4
+	github.com/lib/pq v1.9.0 // indirect
 	github.com/machinebox/graphql v0.2.2
 	github.com/matryer/is v1.4.0 // indirect
 	github.com/onsi/ginkgo v1.14.2
 	github.com/onsi/gomega v1.10.3
 	github.com/opencontainers/runc v1.0.0-rc9 // indirect
 	github.com/pkg/errors v0.9.1
+	github.com/prometheus/client_golang v1.5.1 // indirect
+	github.com/prometheus/common v0.10.0 // indirect
+	github.com/prometheus/procfs v0.2.0 // indirect
 	github.com/robfig/cron v1.2.0
 	github.com/sirupsen/logrus v1.7.0 // indirect
+	github.com/spf13/cobra v1.1.1 // indirect
 	github.com/stretchr/testify v1.6.1
-	golang.org/x/net v0.0.0-20201110031124-69a78807bb2b // indirect
+	golang.org/x/crypto v0.0.0-20201002170205-7f63de1d35b0 // indirect
 	golang.org/x/sys v0.0.0-20201119102817-f84b799fce68 // indirect
-	google.golang.org/grpc v1.28.1
-	gopkg.in/yaml.v2 v2.3.0
-	helm.sh/helm/v3 v3.2.4
-	istio.io/api v0.0.0-20200723170824-3c2193e74947 // indirect
+	golang.org/x/time v0.0.0-20200630173020-3af7569d3a1e // indirect
+	google.golang.org/grpc v1.31.1
+	gopkg.in/yaml.v2 v2.4.0
+	gopkg.in/yaml.v3 v3.0.0-20210107192922-496545a6307b // indirect
+	helm.sh/helm/v3 v3.3.4
+	istio.io/api v0.0.0-20210107192247-a205c627e4b9 // indirect
 	istio.io/client-go v0.0.0-20200128004641-c87542c7dc1d
-	istio.io/gogo-genproto v0.0.0-20191009201739-17d570f95998 // indirect
-	k8s.io/api v0.18.6
-	k8s.io/apimachinery v0.18.6
-	k8s.io/cli-runtime v0.18.4
-	k8s.io/client-go v0.18.6
+	istio.io/gogo-genproto v0.0.0-20210107204948-697d6f912366 // indirect
+	k8s.io/api v0.19.0-alpha.3
+	k8s.io/apiextensions-apiserver v0.19.0-alpha.3 // indirect
+	k8s.io/apimachinery v0.20.1
+	k8s.io/cli-runtime v0.19.0-alpha.3
+	k8s.io/client-go v0.19.0-alpha.3
+	k8s.io/kubectl v0.19.0-alpha.3 // indirect
+	k8s.io/utils v0.0.0-20201110183641-67b214c5f920 // indirect
 	rsc.io/letsencrypt v0.0.3 // indirect
 	sigs.k8s.io/cli-utils v0.20.4
 	sigs.k8s.io/controller-runtime v0.6.2
 	sigs.k8s.io/yaml v1.2.0
 )
 
-replace helm.sh/helm/v3 v3.2.4 => github.com/the-mesh-for-data/helm/v3 v3.2.4-hunchback2
+replace helm.sh/helm/v3 v3.5.0 => github.com/hunchback/helm/v3 v3.5.0-hunchback
```

Comment:
but you have `helm.sh/helm/v3 v3.3.4` in required section. Also did you run `go mod tidy`?

Reply:
I am going to re-run and then run `make verify` to ensure it works